core/drivers/pm/sam/at91_pm.c

```diff
@@ -298,6 +311,7 @@ static TEE_Result at91_enter_backup(void)
 TEE_Result atmel_pm_suspend(uintptr_t entry, struct sm_nsec_ctx *nsec)
 {
 	TEE_Result res = TEE_ERROR_GENERIC;
+	int sctlr = 0;
```

Comment:
This should be of an unsigned type, either `uint32_t` or perhaps `unsigned long`.